asteroid-settings.desktop

```diff
@@ -3,8 +3,8 @@ Type=Application
 Categories=Applications;
 Exec=invoker --single-instance --type=qtcomponents-qt5 /usr/bin/asteroid-settings
 Icon=ios-settings-outline
-X-Asteroid-Center-Color=#4b45b9
-X-Asteroid-Outer-Color=#161537
+X-Asteroid-Center-Color=#0044A6
+X-Asteroid-Outer-Color=#00010C
```

Comment:
Oh I'm surprised this file exists. I don't think it should be there. Desktop files should be generated from the template only.